pyproject.toml

```diff
@@ -0,0 +1,110 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "itk-cuberille"
+version = "2.6.0"
+description = "Convert an ITK Mesh to a simple data structure compatible with vtkPolyData."
+readme = "README.rst"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Dan Meuller" },
+    { name = "Matthew M. McCormick", email = "matt.mccormick@kitware.com" },
+]
+keywords = [
+    "itk",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Intended Audience :: Healthcare Industry",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: Android",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Programming Language :: C++",
+    "Programming Language :: Python",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Topic :: Scientific/Engineering :: Medical Science Apps.",
+    "Topic :: Software Development :: Libraries",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "itk-io == 5.4.*",
+    "numpy",
+]
+
+[project.urls]
+Download = "https://github.com/InsightSoftwareConsortium/ITKCuberille"
+Homepage = "https://github.com/InsightSoftwareConsortium/ITKCuberille"
+
+[tool.scikit-build]
+# The versions of CMake to allow. If CMake is not present on the system or does
+# not pass this specifier, it will be downloaded via PyPI if possible. An empty
+# string will disable this check.
+cmake.version = ">=3.16"
```

Comment:
3.16.3 (is the patch version important?)

Reply:
Not really